sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -1020,7 +1020,9 @@ object SparkSession extends Logging {
       // Return None when running on executors.
       None
     } else {
-      Option(activeThreadSession.get)
+      val session = Option(activeThreadSession.get)
+      session.foreach(_.sparkContext.assertNotStopped())
```

Comment:
I don't think it's necessary. A user may get the active session instance, and after that someone else stops the spark context. This check can't help this case.

Reply:
That's true. This place only checks if user get an active session, existing used session can not be covered. But in some case which used `session.withActive()` this can do the help.